lib/game.rb

```diff
@@ -5,35 +5,54 @@ def initialize
     @board = Board.new
     @players = create_players
     @message_bot = Message.new
+    @turns_taken = 0
+  end
+
+  def start
+    puts @message_bot.welcome
+    play = true   
+    answer = gets.chomp.downcase
+        if answer == 'q'
+            play = false
+        elsif answer == 'p'
+            play
+        else
+          puts @message_bot.invalid_command
+          answer = gets.chomp.downcase
+        end    
+    while play
+        puts @board.print_visual
+        take_turn
+        change_current_player
+        # Move later bros
+        puts @message_bot.last_piece_played(@board.last_piece_played)
```

Comment:
Definitely like moving this under the current player message like Billy suggested